.github/workflows/main.yml

```diff
@@ -75,7 +75,7 @@ jobs:
       - name: Coverage
         run: |
           make coverage
-          sed -i "/^.*internal\/bindata\/cldr\/bindata.go.*$/d" ./cover.out
+          sed -i "/^.*internal\/bindata\/cldr.*$/d" ./cover.out
```

Comment:
Please ignore the folder `internal/bindata`

Reply:
Updated.